common/game_concepts/wc_magic_lifestyles_concepts.txt

```diff
@@ -0,0 +1,68 @@
+﻿wc_magic_lifestyle = {
```

Comment:
The tooltip for not being able to use focus is not specific enough. Was at first very confused why I couldn't use Firespeaking as Anasterian

![bild](https://user-images.githubusercontent.com/4825809/114296414-9849f180-9aab-11eb-8961-83401575b173.png)

Reply:
![bild](https://user-images.githubusercontent.com/4825809/114305377-3ce22880-9ad8-11eb-8105-e6c90db8fe30.png)

Reply:
Added

Reply:
Much better! Good job.